dss/storage/bundles.py

```diff
@@ -119,3 +120,41 @@ def _latest_version_from_object_names(object_names: typing.Iterator[str]) -> str
             version = current_version
 
     return version
+
+
+def list_available_uuids(replica: str = None,
+                         prefix: typing.Optional[str] = None,
+                         per_page: int = PerPageBounds.per_page_max,
+                         search_after: typing.Optional[str] = None,
+                         token: typing.Optional[str] = None):
```

Comment:
This method does not return a list of `uuid`s. Please think of a more descriptive name.

Reply:
complete